### DISCUSSION TITLE
AWS sagemaker notebook doesn't allow sudo insall and other insall for  ffmpeg CLI  < FileNotFoundError: [Errno 2] No such file or directory: 'ffmpeg' >

### DISCUSSION THREAD

**Discussion by halzihaozhou** [openai/whisper#579] [Q&A]
Hi,
   I'm working sagemaker notebook and switch between sagemaker notebook and cloud9, unlike google colab,  they don't have ffmpeg CLI preinstalled,they don't recognize the sudo syntax both in cell and terminal, is there any way to resolve this issue ?
   
   
![image](https://user-images.githubusercontent.com/57652819/203407876-0a7a9718-8c91-4691-a082-c12b12f75920.png)

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/57652819/203408670-39e08019-2020-4477-a484-ebc980006f9f.png)

**Comment by mllife** [comment]
### try this; (in terminal)

pip install ffmpeg

### additional steps for python >= 3.9

wget http://ciscobinary.openh264.org/libopenh264-2.1.1-linux64.6.so.bz2

bunzip2 libopenh264-2.1.1-linux64.6.so.bz2

cp libopenh264-2.1.1-linux64.6.so ~/anaconda3/envs/pytorch_p39/lib/libopenh264.so.5

### verify the installation by running (in notebook)

! ffmpeg
